andela/magma-backend
# #167749950 User email verification

#### What does this PR do?
- It verifies users email

#### Description of task to be completed
- Have verifyEmail function that sends mail to a user
- Have api/v1/verifyEmail/:token route that user reach when clicking the link 
- It updates isVerified column to true

#### How should this be manually tested?
After cloning the repo cd into it and run npm install and npm test
- using the postman test 
- make signup request with the firstname, lastname, email, password.
- check the mail and click the link.

Key: content-type value: application/json

Key: Authorization value: JWT TOKEN

#### What are the relevant pivotal tracker stories
[#167749950](https://www.pivotaltracker.com/story/show/167749950)
